src/main/kotlin/com/github/biomejs/intellijbiome/listeners/BiomeEditorPanelListener.kt

```diff
@@ -0,0 +1,39 @@
+package com.github.biomejs.intellijbiome.listeners
+
+import com.github.biomejs.intellijbiome.BiomePackage
+import com.github.biomejs.intellijbiome.services.BiomeServerService
+import com.github.biomejs.intellijbiome.settings.BiomeSettings
+import com.intellij.openapi.components.service
+import com.intellij.openapi.fileEditor.FileEditorManagerEvent
+import com.intellij.openapi.fileEditor.FileEditorManagerListener
+import com.intellij.openapi.project.Project
+
+class BiomeEditorPanelListener(private val project: Project) : FileEditorManagerListener {
```

Comment:
I don't know much about JetBrains plugins. Would you mind documenting this class and explains what's its role?

Reply:
Added.
It listens for file selection changes in IDE and restarts LSP server if selected file in editor should be checked with different Biome config.